src/bin/pg_autoctl/keeper_pg_init.c

```diff
@@ -709,6 +709,17 @@ create_database_and_extension(Keeper *keeper)
 	/* publish our new pgSetup to the caller postgres state too */
 	postgres->postgresSetup.ssl = initPostgres.postgresSetup.ssl;
 
+	/*
+	 * Ensure pg_stat_statements is installed in the server extension dir used
+	 * to create the Postgres instance
+	 */
+	config_find_pg_ctl(&config->pgSetup);
```

Comment:
Can you explain why you needed to add a call to that function here? This is supposed to be done when parsing the command line arguments, or when reading our configuration file when it exists already. It's quite surprising to see that call here, smells like “modularity violation” kind of thing.

Reply:
Ah, sorry that was a remnant from my previous implementation. In my previous implementation's call site, `pgSetup` didn't have the `pg_ctl` field set.
Thanks for spotting this!